docs/cmd/meroxa_create_resource.md

```diff
@@ -0,0 +1,44 @@
+## meroxa create resource
```

Comment:
I don't think this should have been generated as part of this PR. I wonder if this was generated by accident while refactoring since I don't see anything in the code that should be causing this now.